src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component, OnInit } from '@angular/core';
 import { MatDialogRef } from '@angular/material/dialog';
 import { Router } from '@angular/router';
+import { CreateEcoNewsService } from '../../../../service/eco-news/create-eco-news.service'
 
```

Comment:
![image](https://user-images.githubusercontent.com/54113846/82760839-f5d74000-9dfe-11ea-8c58-cbb6ab012f1c.png)
Bug 693 is still reproducible with invalid image extension. Please, fix it

Reply:
fixed